plugins/generic/usageStats/templates/usageStatsSettingsForm.tpl

```diff
@@ -46,6 +46,14 @@
 			{fbvElement type="checkboxgroup" id="optionalColumns" from=$optionalColumnsOptions selected=$selectedOptionalColumns translate=false}
 		{/fbvFormSection}
 	{/fbvFormArea}
+
+	<h3>
+	{if $currentContext}
+		{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide"}
+	{else}
+		{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide"}
+	{/if}
+	</h3>
```

Comment:
Move the note into the `{fbvFormArea}` and use a `<p>` or `<div>` element instead, ie:

```
{fbvFormArea id="usageStatsDisplayOptions" title="plugins.generic.usageStats.settings.statsDisplayOptions"}
	<p>
		{if $currentContext}
			{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide"}
		{else}
			{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide"}
		{/if}
	</p>
	{fbvFormSection for="displayStatistics" list=true}
```